sdk/communication/azure-communication-networktraversal/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 1.1.1 (2024-04-03)
```

Comment:
If the date is in the past, I don't think the tool will accept it. Also try deleting the 1.1.0b2 unreleased version below if that never had a release.